numba/cuda/cudadrv/devicearray.py

```diff
@@ -331,6 +331,13 @@ def view(self, dtype):
             gpu_data=self.gpu_data,
             )
 
+    @property
+    def nbytes(self):
```

Comment:
This API addition needs reflecting in the CUDA simulator equivalent class `numba.cuda.simulator.cudadrv.devicearray.FakeCUDAArray`.